bpf/flow.h

```diff
@@ -30,8 +32,17 @@ typedef struct flow_metrics_t {
     // 0 otherwise
     // https://chromium.googlesource.com/chromiumos/docs/+/master/constants/errnos.md
     u8 errno;
+    struct tcp_drops_t {
```

Comment:
wouldn't it be more consistent to move `tcp_drops_t` in `tcp_drops.h` ?

Reply:
ah sorry I just realize it's nested

Reply:
also all structures that needs go eqv need to be in the same .h file